### PR TITLE
[OTA] Update README wth a more detailed multiple fabrics example

### PR DESCRIPTION
**Pull request by carol-apple**
#### Problem
There has been some confusion as to how to execute some test plans when it comes to multiple fabrics and OTA

#### Change overview
Add a detailed example of how to commission two providers onto two fabrics and one requestor onto each of those fabrics

#### Testing
No functional change. Tree compiles.

### CODE REVIEW

**Review by isiu-apple — COMMENTED** [review]
`Every 60 seconds from when ...` - Is this referring to the periodic query timer?  Why 60 seconds?

**Comment by carol-apple** [comment]
> `Every 60 seconds from when ...` - Is this referring to the periodic query timer? Why 60 seconds?

This is listed under the example for triggering via the DefaultOTAProviders attribute which requires that a periodic query timer be used. 60 seconds was arbitrarily picked so that if someone were to follow the example, they weren't waiting too long before seeing the trigger occur. That sentence was only there as an explanation for the sequence of commands listed in the above example.

File: examples/ota-requestor-app/linux/README.md [hunk 1]
```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
 not be more than one entry containing the same fabric.
 
 To add more than one entry to the `DefaultOTAProviders` attribute, the OTA
-Requestor app must be commissioned on multiple fabrics. This can be done as
-specified below:
+Requestor app must be commissioned on multiple fabrics. At least one OTA
+Provider app should be commissioned to each corresponding fabric that the OTA
+Requestor app had been commissioned to.
```
Comment (bzbarsky-apple) [inline examples/ota-requestor-app/linux/README.md:324]: Please be consistent about "commissioned on" vs "commissioned to".  I think "commissioned on" is probably better....  Or "comissioned into"?

File: examples/ota-requestor-app/linux/README.md [hunk 2]
```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
...
 
-**Build and run the OTA Requestor application**
+The following example has two OTA Provider apps, each commissioned on a
+different fabric (alpha and beta) and one OTA Requestor app commissioned to both
```
Comment (bzbarsky-apple) [inline examples/ota-requestor-app/linux/README.md:327]: Again, "on" vs "to".

File: examples/ota-requestor-app/linux/README.md [hunk 3]
```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
...
+**Commission the OTA Requestor application to the second fabric (beta)**
+
+```
+out/chip-tool pairing onnetwork-long 0xB0B 20202021 18 --commissioner-name beta
+```
+
+**Write/Read DefaultOTAProviders on the first fabric (alpha)**
+
+```
+out/chip-tool otasoftwareupdaterequestor write default-ota-providers '[{"fabricIndex": 1, "providerNodeID": 12648430, "endpoint": 0}]' 0xDEB 0
```
Comment (bzbarsky-apple) [inline examples/ota-requestor-app/linux/README.md:401]:
```suggestion
out/chip-tool otasoftwareupdaterequestor write default-ota-providers '[{"providerNodeID": 12648430, "endpoint": 0}]' 0xDEB 0
```
The fabricIndex value is ignored anywayand does not need to be provided.

File: examples/ota-requestor-app/linux/README.md [hunk 4]
```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
...
+out/chip-tool otasoftwareupdaterequestor read default-ota-providers 0xDEB 0
 ```
 
-**Write/Read DefaultOTAProviders on second fabric**
+**Write/Read DefaultOTAProviders on second fabric (beta)**
 
 ```
-out/chip-tool otasoftwareupdaterequestor write default-ota-providers '[{"fabricIndex": 2, "providerNodeID": 1, "endpoint": 0}]' 0x858 0 --commissioner-name beta
-out/chip-tool otasoftwareupdaterequestor read default-ota-providers 0x858 0 --commissioner-name beta
+out/chip-tool otasoftwareupdaterequestor write default-ota-providers '[{"fabricIndex": 2, "providerNodeID": 45242, "endpoint": 0}]' 0xB0B 0 --commissioner-name beta
```
Comment (bzbarsky-apple) [inline examples/ota-requestor-app/linux/README.md:408]:
```suggestion
out/chip-tool otasoftwareupdaterequestor write default-ota-providers '[{"providerNodeID": 45242, "endpoint": 0}]' 0xB0B 0 --commissioner-name beta
```

File: examples/ota-requestor-app/linux/README.md [hunk 5]
```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
...
+out/chip-tool otasoftwareupdaterequestor read default-ota-providers 0xB0B 0 --commissioner-name beta
 ```
+
+**Write ACL for the first OTA Provider application**
+
+```
+out/chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": null, "targets": [{"cluster": 41, "endpoint": null, "deviceType": null}]}]' 0xC0FFEE 0
```
Comment (bzbarsky-apple) [inline examples/ota-requestor-app/linux/README.md:415]:
```suggestion
out/chip-tool accesscontrol write acl '[{"privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"privilege": 3, "authMode": 2, "subjects": null, "targets": [{"cluster": 41, "endpoint": null, "deviceType": null}]}]' 0xC0FFEE 0
```

File: examples/ota-requestor-app/linux/README.md [hunk 6]
```diff
@@ -319,48 +319,108 @@ structs. Each entry in this list is a default OTA Provider per fabric. There can
...
+```
+
+**Write ACL for the second OTA Provider application**
+
+```
+out/chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": null, "targets": [{"cluster": 41, "endpoint": null, "deviceType": null}]}]' 0xB0BA 0 --commissioner-name beta
```
Comment (bzbarsky-apple) [inline examples/ota-requestor-app/linux/README.md:421]:
```suggestion
out/chip-tool accesscontrol write acl '[{"privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"privilege": 3, "authMode": 2, "subjects": null, "targets": [{"cluster": 41, "endpoint": null, "deviceType": null}]}]' 0xB0BA 0 --commissioner-name beta
```

**Review by bzbarsky-apple — APPROVED** [review]

**Review by selissia — APPROVED** [review]

File: examples/ota-requestor-app/linux/README.md (hunk 1 above)
Comment (carol-apple) [reply examples/ota-requestor-app/linux/README.md]: Used "commissioned into" throughout

**Pull request merged**